tests/unit/src/test/scala/tests/DefinitionLspSuite.scala

```diff
@@ -326,4 +326,67 @@ class DefinitionLspSuite extends BaseLspSuite("definition") {
     } yield ()
   }
 
+  test("clashing-references") {
```

Comment:
You should also uncomment https://github.com/scalameta/metals/blob/03704c17b5397bd8feb4771d0bedf9a6dfe0a660/tests/unit/src/test/scala/tests/debug/BreakpointDapSuite.scala#L633
to see if it's working correctly now.

Reply:
Thx! Now it works.